wrangler.toml

```diff
@@ -34,3 +35,4 @@ migrations_dir = "migrations"
 
 [env.production.vars]
 DELETE_OLD_DATA_BEFORE = "-3 months" # Standard SQL format
```

Comment:
Do you have a link for the format that goes here? I'm not familiar

Reply:
https://www.sqlite.org/lang_datefunc.html#modifiers